boltdesignsystem/bolt
# DS-690: Remove Nightwatch

## Jira

https://pegadigitalit.atlassian.net/browse/DS-690

## Summary

Removed Nightwatch from the build.

## Details

* Removed Nightwatch tasks from the main `package.json` file
* Removed the Nightwatch tasks from the `travis.xml` file
* Deleted all the `.e2e.js` files within the `/packages/components` folder
* Removed Nightwatch dependencies
* Removed the `packages/testing/ttesting-nightwatch` folder

## How to test

1. Review the code
2. Confirm that Travis has successfully ran (with no Nightwatch tasks running)
3. Confirm that Travis has successfully ran on this https://github.com/boltdesignsystem/bolt/pull/2431 (with no Nightwatch tasks running)
